pkg/graph/resolvers/resolver_test_utilities.go

```diff
@@ -135,14 +135,15 @@ func createTemplateCacheHelper(
 	planName string,
 	plan *models.ModelPlan) (*emailtemplates.EmailTemplate, string, string) {
 
-	return createTemplateCacheHelperWithInputTemplates(
+	return CreateTemplateCacheHelperWithInputTemplates(
 		planName,
 		plan,
 		"{{.ModelName}}'s Test",
 		"{{.ModelName}} {{.ModelID}}")
 }
 
-func createTemplateCacheHelperWithInputTemplates(
+// CreateTemplateCacheHelperWithInputTemplates creates a test template with the given subject and body
+func CreateTemplateCacheHelperWithInputTemplates(
```

Comment:
Nit: I wonder if this shoud live in another package now that it is being exported? Perhaps `emailtestconfigs`? No need to move for this PR